cfep-10.md

```diff
@@ -0,0 +1,59 @@
+
+<table>
+<tr><td> Title </td><td> Feedstock Statuses </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Christopher J. "CJ" Wright &lt;cjwright4242@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 29, 2019</td></tr>
+<tr><td> Discussion </td><td> link to the PR where the CFEP is being discussed, NA is circulated initially </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not availble </td></tr>
+</table>
+
+## Abstract
+
+This document proposes the creation of four new statuses for feedstocks that denote the	
+standing of the feedstock and transistions between the standings.
+These statuses will help us triage feedstocks and identify which feedstocks need
+help, maintainers, or archiving.
+
+## Motivation
+
+It can be difficult for maintainers and would be maintainers to identify which feedstocks could use their
+help.
+Additionally, it is difficult for core to know which feedstocks need additional help or archiving.
+Finally, users of CF packages may find it difficult to understand which packages are dorment.
+This proposal aims to expose this information more clearly.
```

Comment:
The last sentence of this paragraph does not seem strictly correct. This proposal has very little to say about *exposing* status information at the moment.